Correct Getting Started instructions (#5713)
activeadmin/activeadmin: docs/documentation.md

$> rails server
```

> Visit `http://localhost:3000/admin` and voila: You&#8217;re on your brand

Reviewer: Do we need the HTML character here? Also, I would avoid the french word

```suggestion
Visit `http://localhost:3000/admin` and you'll be on your brand
```

new Active Admin dashboard.

To register an already existing model, run:

> 

Reviewer: I just noticed that the "Voila!", and the HTML characters were already there, thanks for updating that even if it was not introduced by you :+1:.

Author: :-)

```bash
$> rails generate active_admin:resource [MyModelName]
```

This creates a file at `app/admin/my_model_names.rb` for configuring the
resource. Refresh your web browser to see the interface. In order to CRUD

> items, tweak the parameter `permit_params` in `app/admin/MyModel.rb`.

Reviewer: ```suggestion
items, tweak `permit_params` in `app/admin/my_model_names.rb`.
```

## Usage with devise

> If you want to use the devise gem for (admin) authentification, add it to

Reviewer: I missed this instance in the previous review

```suggestion
If you want to use the devise gem for (admin) authentication, add it to
```

your Gemfile

```ruby

> # Gemfile

Reviewer: We already mentioned the Gemfile above so we probably don't need this comment?

```suggestion
```

gem 'activeadmin'
gem 'devise'
```

> Update the bundle and run the installer

Reviewer: Maybe "Update the bundle" will not be clear for some users? What about

```suggestion
Run `bundle install`, and then run the installer
```

```bash

> rails generate active_admin:install

Reviewer: I don't love the existing style, but let's keep it as it is for now

```suggestion
$> rails generate active_admin:install
```

```

Migrate your db

```bash
$> rake db:migrate
```

If you are adding devise with Active Admin, you need to seed the database
with an admin user for Active Admin (otherwise you probably will already

> have an admin user)

Reviewer: I would clarify here that the installer adds an admin user to the `seeds.rb` file, and that's it's up to the user whether to reseed the whole thing or manually create the admin user.
